src/lib.rs

```diff
@@ -1219,6 +1301,70 @@ pub fn handle_compile_subcommand(
     Ok(json!({"descriptor": descriptor.to_string()}))
 }
 
+/// Proof of reserves verification sub-command
+///
+/// Proof of reserves options are described in [`CliSubCommand::Reserves`].
+#[cfg(all(feature = "reserves", feature = "electrum"))]
+pub fn handle_reserves_subcommand(
```

Comment:
It would be better to name it `handle_ext_reserves_subcommand` or something. The internal reserves command are already handled by `OnlineWalletSubcommand`. So this might be confusing.